plugins/scaffolder/src/next/TemplateEditorPage/TemplateFormPage.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { makeStyles } from '@material-ui/core/styles';
+
+import { Page, Header, Content } from '@backstage/core-components';
+import { useRouteRef } from '@backstage/core-plugin-api';
+import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
+import {
+  FormProps,
+  LayoutOptions,
+  FieldExtensionOptions,
+} from '@backstage/plugin-scaffolder-react';
+
+import { editRouteRef } from '../../routes';
+import { scaffolderTranslationRef } from '../../translation';
+
+import { TemplateFormPreviewer } from './TemplateFormPreviewer';
+
+const useStyles = makeStyles({
+  root: {
+    padding: 0,
+  },
+});
+
+interface TemplateFormPageProps {
+  layouts?: LayoutOptions[];
+  formProps?: FormProps;
+  fieldExtensions?: FieldExtensionOptions<any, any>[];
+  defaultPreviewTemplate?: string;
+}
+
+export function TemplateFormPage(props: TemplateFormPageProps) {
+  const classes = useStyles();
+  const navigate = useNavigate();
+  const editLink = useRouteRef(editRouteRef);
+  const { t } = useTranslationRef(scaffolderTranslationRef);
+
+  const handleClose = useCallback(() => {
+    navigate(editLink());
+  }, [navigate, editLink]);
+
+  return (
+    <Page themeId="home">
+      <Header
+        title={t('templateEditorPage.title')}
```

Comment:
Let's change this tile to Template Form Playground or something similar.

Might also add a breadcrumb back to either /create or /edit

```
        type="scaffolder"
        typeLink={createLink()}
```

Reply:
Done 🙌🏻